integration-tests/src/test/java/org/apache/druid/tests/query/ITSqlQueryTest.java

```diff
@@ -350,15 +326,13 @@ public void testMultipleContentType()
   {
     executeQuery(
         MediaType.TEXT_PLAIN,
+        "SELECT 1",
         (request) -> {
```

Comment:
With this patch, in all the other cases, the `request` object seems to remain unchanged.
In this method, we just add an extra header.
So maybe instead of passing lambda `OnRequest` to the method `executeQuery`, we just pass a map of headers for simplicity.